app/mailers/audit_mailer.rb

```diff
@@ -0,0 +1,19 @@
+class AuditMailer < ApplicationMailer
+  helper :audits, :layout
+
+  def summary(options = {})
+    raise ::Foreman::Exception.new(N_("Must specify a user with email enabled")) unless (user=User.find(options[:user])) && user.mail_enabled?
+    time     = options[:time] ? %(time > "#{options[:time]}") : 'time > yesterday'
+    set_url
+    set_locale_for(user)
+    @query    = options[:query] ? "#{options[:query]} and #{time}" : "#{time}"
+    @limit   = options[:limit] || SETTINGS[:entries_per_page]
```

Comment:
should be `Setting[:entries_per_page]` as it's a db Setting, not a SETTING(.yaml).